### DISCUSSION TITLE
Nothing happens when I click on "Start on boot".

### DISCUSSION THREAD

**Discussion by zekuraz** [rustdesk/rustdesk#8927] [Q&A]
Hi !

We have a fleet of Viewsonic CDE5520 and CDE4320 screens.

We'd like to control these screens remotely. To do this, we have installed the RustDesk Client software (https://rustdesk.com/docs/en/client/). The software works. However, to save energy, we have set up a system that puts the screens to sleep. As soon as the screens are restarted, the RustDesk Client software services do not restart automatically. So you have to move around manually and restart the software, which is annoying.

I noticed that the application has a "Start on Boot" feature that requires special permissions (see attachment). However, when I click on it, nothing happens. I wanted to know if there is a procedure for enabling these permissions on android.

(Another remark, we also have CDE6520, and on this model it works perfectly with the restart of the RustDesk Client service. I access the special permissions on this model) 

thank you in advance for your clarifications

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#android-start-on-boot-and-limitations
any further discussion please go to https://github.com/rustdesk/rustdesk/discussions/1330
